pylint/extensions/while_used.py

```diff
@@ -24,7 +24,8 @@ class WhileChecker(BaseChecker):
         "W0149": (
             "Used `while` loop",
             "while-used",
-            "Unbounded `while` loops can often be rewritten as bounded `for` loops.",
+            "Unbounded `while` loops can often be rewritten as bounded `for` loops. ",
+            "Exceptions can be made for cases such as event loops, listeners, etc.",
```

Comment:
I think the added tuple item is an issue here. Mypy is highlighting it in the precommit output.